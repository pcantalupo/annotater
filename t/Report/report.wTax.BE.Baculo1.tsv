seqID	seq	seqLength	pid	coverage	e	accession	desc	type	family	species	genome	algorithm	db	qstart	qend	sstart	ssend	nsf	qent	qhsp_ent	shsp_ent	shsp_%lc
contig_A	CAA	3	100	100	1.00E-05	gi|1226168|gb|AAA92249.1|	ORF B (bases 1850-5560)	virus	Baculoviridae	Autographa californica nucleopolyhedrovirus	dsDNA,circular,nonsegmented	rapsearch	nr	537	7	650	875	1	85	84	-1	-1
contig_B	CGA	3	100	100	1.00E-05	gi|1226168|gb|AAA92249.1|	ORF B (bases 1850-5560)	virus	Baculoviridae	Autographa californica nucleopolyhedrovirus	dsDNA,circular,nonsegmented	rapsearch	nr	1000	200	680	849	1	85	84	-1	-1
contig_C	TTA	3	100	100	1.00E-05	gi|1226168|gb|AAA92249.1|	ORF B (bases 1850-5560)	virus	Baculoviridae	Autographa californica nucleopolyhedrovirus	dsDNA,circular,nonsegmented	rapsearch	nr	800	300	671	820	1	85	84	-1	-1
