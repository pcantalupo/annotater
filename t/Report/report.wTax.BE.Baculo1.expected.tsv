contig_A	CAA	3																1	85	84	-1	-1
contig_B	CGA	3																1	85	84	-1	-1
contig_C	TTA	3																1	85	84	-1	-1
